jobs/build/multi_promote/Jenkinsfile

```
@@ -5,7 +5,9 @@ node {
     def release = load("pipeline-scripts/release.groovy")
     def buildlib = release.buildlib
     def commonlib = release.commonlib
-    commonlib.describeJob("multi_promote", """
+    commonlib.describeJob("⛔❌⛔ multi_promote (deprecated) ⛔❌⛔ ", """
+        <b>Deprecated. Use <a href="https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fpromote-assembly/">promote-assembly</a> instead</b>
```

personally prefer not to hardcode anything we don't have to:
```suggestion
        <b>Deprecated. Use <a href="/job/aos-cd-builds/job/build%252Fpromote-assembly/">promote-assembly</a> instead</b>
```